FHEM/00_SIGNALduino.pm

```diff
@@ -228,7 +228,7 @@ SIGNALduino_Initialize($)
 					  ." cc1101_frequency"
 					  ." doubleMsgCheck_IDs"
 					  ." suppressDeviceRawmsg:1,0"
-					  ." development"
+					  ." development:0,1"
```

Comment:
Vielleicht sollten wir hier noch einbauen

`if (SDUINO_VERSION   =~ m/.*dev_.*/ )?         `

Reply:
Ja, ist eine sehr gute Idee
So müsste es ausreichen. Mit `." development:0"` ist das aktivieren nicht mehr so einfach möglich.
Und ein debug macht für mich in der master auch keinen Sinn

```
  my $dev = "";
  if (index(SDUINO_VERSION, "dev") >= 0) {
     $dev = ",1";
  }

." debug:0$dev"
." development:0$dev"
```

Reply:
Da es das Attribut Development im master nicht gibt, würde ich es auch überhaupt nicht anbieten. Auch nicht mit dem Wert 0.

Reply:
Ich habe die saubere Lösung gewählt, wie von betateilchen vorgeschlagen.
Das Attribut development wird ignoriert und es gibt eine Logausgabe

Reply:
die saubere Lösung ist also ein Attribut im Master anzubieten, welches dort nicht gesetzt werden kann?

Reply:
sauberer als die Fehlermeldung mit einem Eintrag in motd, wo wir Beiträge im Forum bekommen.

Reply:
Du meint, wenn jemans von dev-r33 auf den Master wechselt?

Reply:
hmm, nagut. Die perfekte Lösung gibt es nicht. Dann machen wir es halt erst mal so.
In einem anderen PR können wir ja eventuell Hinweise im Modul selbst geben, dass das Attribut gelöscht werden sollte.

Reply:
Der Hinweis (im Forum und im wiki) sollte am Besten von Dir kommen, wenn es eine neue Version im master und svn gibt.